lib/azure/armrest/resource_provider_service.rb

```diff
@@ -53,7 +54,8 @@ def list
       def get(namespace)
         url = build_url(namespace)
         response = rest_get(url)
-        JSON.parse(response.body)
+        #JSON.parse(response.body)
+        Azure::Armrest::ResourceProvider.new(response.body)
```

Comment:
The plan is to eventually replace the JSON.parse calls with instances of our new wrapper classes. I thought I would make those changes as a separate PR.